modules/ROOT/pages/dialog-components.adoc

```diff
@@ -539,6 +539,96 @@ A *textarea* is a multiline text field.
 }
 ----
 
+[[tree]]
+=== tree
+
+A *tree* is a composite component that presents an expandable category tree expandable down one level. It can also include a search and filter field.
```

Comment:
This is not accurate. This description fits the advtemplate plugin.
The tree component does NOT include a search and filter field.
Also the tree component does NOT present expandable category, and it is not expandable down ONLY one level.

A more accurate description in my opinion would be:
A tree is a component that can be used to represent a tree hierarchy such as folders and files for example. 

A tree can have two types of tree items, a directory tree item and a leaf tree item.
A directory tree item can contain leaf items or other directory items.
Clicking on a directory toggles its expanding/collapsing state.
Clicking on a leaf item triggers the onLeafAction callback.

I apologize if I didn't make this wasn't clear before.

Reply:
This introduction has been completely re-written.